src/components/ADempiere/Form/VPOS/Collection/index.vue

```diff
@@ -805,6 +818,16 @@ export default {
         paymentUuid
       })
     },
+    validateOrder(payment) {
+      this.porcessInvoce = true
+      if (this.pay > this.currentOrder.grandTotal) {
+        this.$store.commit('dialogoInvoce', { show: true, type: 1 })
+      } else if (this.pay < this.currentOrder.grandTotal) {
+        this.$store.commit('dialogoInvoce', { show: true, type: 2 })
```

Comment:
please change state name to `dialogInvoice` (without the `o` at the end of the dialog)